### DISCUSSION TITLE
update linux，The console  output appears unreadable code

### DISCUSSION THREAD

**Discussion by yeliangchen2** [polarfire-soc/polarfire-soc-documentation#232] [Linux drivers for PolarFire Soc] [answered]
linux kernel version
![image](https://user-images.githubusercontent.com/16462894/198819495-a2489a80-9624-4123-8e36-9be1c4745281.png)

![image](https://user-images.githubusercontent.com/16462894/198819567-ca35e2ef-503f-4fed-af9a-8c3f89bff7aa.png)

**Comment by hughbreslin** [comment, marked as answer]
Hi @yeliangchen2 - its hard to debug this or provide a useful answer without much information. 

Is this on an Icicle Kit? Why are you using the 2022.10-rc3 L4M kernel and what version were you using before you updated? 

If this isn't an Icicle Kit and you have an MSS reference clock that isn't 125MHz you will need to update the reference clock frequency in the device tree, for example on the Icicle Kit it is [here](https://github.com/linux4microchip/linux/blob/linux-5.15-mchp/arch/riscv/boot/dts/microchip/mpfs-icicle-kit.dts#L196). An incorrect frequency could cause this issue.
